### DISCUSSION TITLE
Importação para Roll20

### DISCUSSION THREAD

**Discussion by DoctorBlind** [YuriAlessandro/gerador-ficha-tormenta20#318] [Ideias]
Vi a possibilidade da criação de arquivos Json para roll20 pela existência do VTTES (Extensão) Importer o formato é bem menos complexo que o aceito pelo Foundry e auxiliaria muito pois não existe ficha compacta para npcs ou inimigos, então tem que criar do zero toda a ficha ou usar macro o que é um saco kaskdk.

Em anexo o formato json do VTTES
[Ficha modelo.zip](https://github.com/YuriAlessandro/gerador-ficha-tormenta20/files/7689092/Ficha.modelo.zip)

**Comment by YuriAlessandro** [comment]
Boa sugestão. Acho que a gente não criou de primeira porque a gente não costuma usar o Roll20, mas com a ficha modelo a gente deve conseguir bolar algo. 

Vou adicionar na lista de coisas a serem feitas e dou um retorno quando finalizarmos.

**Comment by LeonardoCalazans** [comment]
Eu pensei exatamente a mesma coisa, achei esse projeto procurando personagens prontos para a minha campanha. Queria usar uns inimigos diferenciados. Sensacional esse projeto, parabéns.
